Rubberduck.Refactorings/ImplementInterface/ImplementInterfaceRefactoring.cs

```diff
@@ -125,126 +129,68 @@ private void AddItems(List<Declaration> missingMembers, IModuleRewriter rewriter
+                    {
+                        members.Add(string.Format(template, "Property", " Let", signature, string.Empty));
+                    }
 
-            return string.Join(Environment.NewLine, memberSignature, MemberBody, memberCloseStatement);
-        }
+                    if (member.AsTypeName.Equals(Tokens.Variant) || member.IsObject)
+                    {
+                        members.Add(string.Format(template, "Property", " Set", signature, string.Empty));
+                    }
 
-        private List<Parameter> GetParameters(Declaration member)
-        {
-            var parameters = _declarations.Where(item => item.DeclarationType == DeclarationType.Parameter &&
-                              item.ParentScopeDeclaration == member)
-                           .OrderBy(o => o.Selection.StartLine)
-                           .ThenBy(t => t.Selection.StartColumn)
-                           .Select(p => new Parameter
-                           {
-                               Accessibility = ((VBAParser.ArgContext)p.Context).BYVAL() != null
-                                            ? Tokens.ByVal 
-                                            : Tokens.ByRef,
-
-                               Name = p.IdentifierName,
-                               AsTypeName = p.AsTypeName
-                           })
-                           .ToList();
-
-            return parameters;
-        }
+                    return string.Join(Environment.NewLine, members);
+            }
 
-        private IEnumerable<Declaration> GetInterfaceMembers()
-        {
-            return _declarations.FindInterfaceMembers()
-                                .Where(d => d.ComponentName == _targetInterface.IdentifierName)
-                                .OrderBy(d => d.Selection.StartLine)
-                                .ThenBy(d => d.Selection.StartColumn);
+            return string.Empty;
         }
 
-        private IEnumerable<Declaration> GetImplementedMembers()
+        private IEnumerable<Parameter> GetParameters(Declaration member)
         {
-            return _declarations.FindInterfaceImplementationMembers()
-                                .Where(item => item.ProjectId == _targetInterface.ProjectId
-                                        && item.ComponentName == _targetClass.IdentifierName
-                                        && item.IdentifierName.StartsWith(_targetInterface.ComponentName + "_")
-                                        && !item.Equals(_targetClass))
-                                .OrderBy(d => d.Selection.StartLine)
-                                .ThenBy(d => d.Selection.StartColumn);
-        }
+            if (member.DeclarationType == DeclarationType.Variable)
+            {
+                return new List<Parameter>
+                {
+                    new Parameter
+                    {
+                        Accessibility = Tokens.ByVal,
```

Comment:
`Accessibility` here, is conflating a declaration's accessibility with whether the parameter is passed as a reference or as a copy. Accessibility can only ever be `Public`, `Private` or `Friend`. It can't be `ByVal` or `ByRef`, and it can't be `Implicit` or `Global` either

Reply:
See above.